### ISSUE TITLE
PopupBridge 2.0.0 incompatible with Browser Switch 1.0.0

### ISSUE THREAD

**Issue by ytilliette** [braintree/popup-bridge-android#31]
### General information

* SDK/Library version: 2.0.0
* Android Version and Device: All

### Issue description

After updating the Braintree SDK to 3.13 the app crashes when callling `PopupBridge.newInstance()`
Looks like the Browser Switch SDK version that was updated with the latest Braintree SDK is not compatible with the PopupBridge.

```
2020-09-02 10:27:21.074 31626-31626/com.fandango E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.fandango, PID: 31626
    java.lang.NoSuchFieldError: No field mContext of type Landroid/content/Context; in class Lcom/braintreepayments/popupbridge/PopupBridge; or its superclasses (declaration of 'com.braintreepayments.popupbridge.PopupBridge' appears in /data/app/com.******-m0me19VifHzCFlA14iHs-Q==/base.apk)
        at com.braintreepayments.popupbridge.PopupBridge.getReturnUrlScheme(PopupBridge.java:158)
        at com.braintreepayments.browserswitch.BrowserSwitchFragment.onCreate(BrowserSwitchFragment.java:23)
        at com.braintreepayments.popupbridge.PopupBridge.onCreate(PopupBridge.java:95)
        at androidx.fragment.app.Fragment.performCreate(Fragment.java:2685)
        at androidx.fragment.app.FragmentStateManager.create(FragmentStateManager.java:280)
        at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1187)
        at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1368)
        at androidx.fragment.app.FragmentManager.moveFragmentToExpectedState(FragmentManager.java:1446)
        at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1509)
        at androidx.fragment.app.BackStackRecord.executeOps(BackStackRecord.java:447)
        at androidx.fragment.app.FragmentManager.executeOps(FragmentManager.java:2181)
        at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:2004)
        at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:1959)
        at androidx.fragment.app.FragmentManager.execSingleAction(FragmentManager.java:1830)
        at androidx.fragment.app.BackStackRecord.commitNow(BackStackRecord.java:297)
        at com.braintreepayments.popupbridge.PopupBridge.newInstance(PopupBridge.java:66)
```

**Comment by sarahkoop** [comment]
Hi @ytilliette, thanks for using the Braintree Android SDK. We are working on a fix for this issue. In this instance, the browser-switch dependency for the core SDK is overriding the dependency version in popup-bridge. We will update here when we have a fix.

**Comment by sshropshire** [comment]
Hey @ytilliette we have a fix and will push a new release of `popup-bridge` once we get one more item approved.

**Comment by sshropshire** [comment]
@ytilliette we have released `popup-bridge` v3.0.0 that should resolve this issue.

**Comment by ytilliette** [comment]
It's not crashing anymore.
Thank you very much @sarahkoop and @sshropshire !
